### PR TITLE
Email validation

### PR DESCRIPTION
**Pull request by shrikrishnaholla**
Added email validation in Create and Update actions in org and users

### CODE REVIEW

File: api/orgs/create.js [hunk 1]
```diff
@@ -7,11 +9,16 @@ module.exports = function (app, db) {
     // Check for required params
     var errs = app.utils.need(req, ['name', 'email']);
     var name = req.body.name;
+    var email = req.body.email;
```
Comment (pksunkara) [inline api/orgs/create.js]: You don't need this line.

File: api/orgs/create.js [hunk 2]
```diff
@@ -7,11 +9,16 @@ module.exports = function (app, db) {
...
 
     if (typeof name != 'string' || name.length < 3 || name.length > 15 || name.match(/[a-z0-9][a-z0-9\ ]*[a-z0-9]/i)[0] != name) {
       errs.push({ field: 'name', code: 'invalid' });
     }
 
+    if (!validator.isEmail(email)) {
```
Comment (pksunkara) [inline api/orgs/create.js]: You can just do `validator.isEmail(req.body.email)`.

File: api/users/create.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 var crypto = require('crypto');
+var validator = require('validator');
```
Comment (pksunkara) [inline api/users/create.js]: Can you change this into the correct format?

```
var crypto = require('crypto')
  , validator = require('validator');
```

File: package.json [hunk 4]
```diff
@@ -12,16 +12,17 @@
     "node": "0.10.x"
   },
   "dependencies": {
-    "express": "~4.0.0",
+    "analytics-node": "1.1.x",
```
Comment (pksunkara) [inline package.json]: Please don't resort the dependencies in this file. I arranged them by priority.

**Comment by pksunkara** [comment]
Awesome work. :+1: Just a few minor style fixes.

**Comment by shrikrishnaholla** [comment]
I don't know why your forbidden_id commit came here. Sorry. How should I fix it? :-/

File: package.json (hunk 4 above)
Comment (shrikrishnaholla) [inline package.json]: This happened when i ran `npm install --save node-validator`.

File: api/users/create.js (hunk 3 above)
Comment (shrikrishnaholla) [inline api/users/create.js]: Done n' pushed. I've seen this in other places too. Why is this the recommended style?

**Comment by pksunkara** [comment]
Merged.

**Pull request closed without merging**